lesson_15/test_romb_figure.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import logging
+
+# Create a logger that writes to a file
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+handler = logging.FileHandler('test_romb_figure.log')
+handler.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+from lesson_15.romb_figure import Rhombus
+
+def test_rhombus_creation():
```

Comment:
I told you several times - if you want to create test cases - you need to define for them Test Suite class. Test methods should not be stored as simple functions.